src/components/Form/EditActivationForm/EditActivationForm.tsx

```diff
@@ -56,10 +56,13 @@ const CreateActivationForm = () => {
     createActivationMutation.mutate({ body: organization });
   };
 
-  const { data, error, isLoading } =
-    remoteData.Backoffice.AttributeAuthority.getOrganization.useQuery({
-      keyOrganizationFiscalCode: operatorFiscalCode
-    });
+  const {
+    data,
+    error,
+    isPending: isLoading
```

Comment:
Consider omitting the renaming and using `isPending` as-is for clarity

```suggestion
    isPending
```